### PR TITLE
doc: working_with_pmic: Added developing with npm1300

### PR DESCRIPTION
**Pull request by aasinclair**
Added developing with npm1300 page.
Includes information on how to use PowerUP overlays.

### CODE REVIEW

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+The |NCS| provides support for development with the `nPM1300 Power Management IC (PMIC) <nPM1300 product website_>`_, using the `nPM1300 EK (PCA10152) <nPM1300 EK product page_>`_.
+
+To get started with your nPM1300 EK, follow the steps in the :ref:`ug_npm1300_gs` section.
+If you are not familiar with the |NCS| and its development environment, see :ref:`installation` and :ref:`configuration_and_build` documentation.
+
+Importing an overlay from nPM PowerUP
+*************************************
+
+nPM PowerUP supports exporting a full configuration of the nPM1300 in devicetree overlay format.
+This exported overlay file can be used to quickly configure the nPM1300 in your application.
```
Comment (peknis) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:19]:
```suggestion
You can use this exported overlay file to quickly configure the nPM1300 in your application.
```

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+If there is no overlay file for you project, include the file directly in your application folder with the name
+`app.overlay`
```
Comment (peknis) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:22]:
```suggestion
:file:`app.overlay`.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.. _ug_npm1300_developing:
+
+Developing with nPM1300
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| provides support for development with the `nPM1300 Power Management IC (PMIC) <nPM1300 product website_>`_, using the `nPM1300 EK (PCA10152) <nPM1300 EK product page_>`_.
```
Comment (annakielar) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:10]:
```suggestion
The |NCS| provides support for development with the `nPM1300 <nPM1300 product website_>`_ Power Management IC (PMIC), using the `nPM1300 EK <nPM1300 EK product page_>`_ (PCA10152).
```

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.. _ug_npm1300_developing:
+
+Developing with nPM1300
+#######################
```
Comment (annakielar) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:4]:
```suggestion
Developing with nPM1300 EK
##########################
```

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| provides support for development with the `nPM1300 Power Management IC (PMIC) <nPM1300 product website_>`_, using the `nPM1300 EK (PCA10152) <nPM1300 EK product page_>`_.
+
+To get started with your nPM1300 EK, follow the steps in the :ref:`ug_npm1300_gs` section.
+If you are not familiar with the |NCS| and its development environment, see :ref:`installation` and :ref:`configuration_and_build` documentation.
```
Comment (annakielar) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:13]:
```suggestion
If you are not familiar with the |NCS| and its development environment, see the :ref:`installation` and :ref:`configuration_and_build` documentation.
```

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+Importing an overlay from nPM PowerUP
+*************************************
+
+nPM PowerUP supports exporting a full configuration of the nPM1300 in devicetree overlay format.
```
Comment (annakielar) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:18]:
```suggestion
nPM PowerUP application supports exporting a full configuration of the nPM1300 in devicetree overlay format.
```
Am I right?

**Review by annakielar — CHANGES_REQUESTED** [review]

**Review by annakielar — APPROVED** [review]

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+The |NCS| provides support for development with the `nPM1300 <nPM1300 product website_>`_ Power Management IC (PMIC), using the `nPM1300 EK <nPM1300 EK product page_>`_ (PCA10152).
+
+To get started with your nPM1300 EK, follow the steps in the :ref:`ug_npm1300_gs` section.
+If you are not familiar with the |NCS| and its development environment, see the :ref:`installation` and :ref:`configuration_and_build` documentation.
+
+Importing an overlay from nPM PowerUP
+*************************************
+
+nPM PowerUP application supports exporting a full configuration of the nPM1300 in devicetree overlay format.
```
Comment (peknis) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:18]:
```suggestion
The nPM PowerUP application supports exporting a full configuration of the nPM1300 in devicetree overlay format.
```

File: doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+You can use this exported overlay file to quickly configure the nPM1300 in your application.
+
+If there is no overlay file for you project, include the file directly in your application folder with the name
```
Comment (peknis) [inline doc/nrf/device_guides/working_with_pmic/npm1300/developing.rst:21]:
```suggestion
If there is no overlay file for your project, include the file directly in your application folder with the name
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**